msal/src/main/java/com/microsoft/identity/client/Logger.java

```diff
@@ -63,8 +63,6 @@
 public final class Logger {
```

Comment:
>Logger [](start = 19, length = 6)

should this class need to have a private constructor to prevent anybody creating an instance of Logger? 

private Logger() {};